Python/pystate.c

```diff
@@ -2977,6 +2977,26 @@ _none_shared(PyThreadState *tstate, PyObject *obj,
     return 0;
 }
 
+static PyObject *
```

Comment:
PEP 7

Reply:
What's wrong specifically?

Reply:
Oh, I'm misread something. Sorry for the false report.